pkg/controller/service/reconciler.go

```diff
@@ -517,7 +517,19 @@ func (s *ServiceReconciler) Reconcile(ctx context.Context, id string) (result re
 		metrics.WithServiceReconciliationStage(metrics.ServiceReconciliationApplyFinish),
 	)
 
-	if err = s.UpdateStatus(svc.ID, svc.Revision.ID, svc.Sha, lo.ToSlicePtr(components), lo.ToSlicePtr(errs)); err != nil {
+	// Extract images metadata from the applied resources
+
+	metadata := s.ExtractImagesMetadata(manifests)
+
+	if metadata != nil {
+		if len(metadata.Images) > 0 {
```

Comment:
we're supposed to also extract fqdns fwiw too (it's in the schema) .  Obvious sources of these are ingress hosts, similar things for gateway api HTTPRoutes etc